hw/ip_templates/rv_plic/fpv/rv_plic_fpv.core.tpl

```diff
@@ -2,28 +2,28 @@ CAPI=2:
 # Copyright lowRISC contributors.
 # Licensed under the Apache License, Version 2.0, see LICENSE for details.
 # SPDX-License-Identifier: Apache-2.0
-name: "lowrisc:fpv:rv_plic_generic_fpv:0.1"
-description: "FPV genenric testbench for RISC-V PLIC example"
+name: ${instance_vlnv("lowrisc:ip:rv_plic_fpv:0.1")}
+description: "FPV for RISC-V PLIC"
 
 filesets:
   files_formal:
     depend:
-      - lowrisc:fpv:rv_plic_component_fpv
```

Comment:
this dependency is now missing. either we add it back, or we include the `vip/rv_plic_assert_fpv.sv` file as a source file below.

Reply:
(As above.)